pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/self-managed/index.md

```diff
@@ -11,6 +11,35 @@ Konvoy deploys all cluster lifecycle services to a bootstrap cluster, which then
 
 Before you start, make sure you have created a workload cluster, as described in [Create the Cluster][createthecluster].
 
+## Explore the cluster
+
+Before checking making the cluster manage itself, explore your cluster:
+
+   ```sh
+   kubectl get pods -A --kubeconfig ${CLUSTER_NAME}.conf
+   ```
+
+<p class="message--note"><strong>NOTE: </strong>If you see a <code>calico-node</code> pod not working on your ready on your cluster, you will have to edit the <code>installation</code> file.
+</p>
+
+If you have to edit the installation file, run the command:
+
+   ```sh 
+   kubectl edit installation default --kubeconfig ${CLUSTER_NAME}.conf
+   ```
+
+Change the `spec.calicoNetwork.nodeAddressAutodetectionV4` to be `interface: ens192`, and save this file:
```

Comment:
I would like to take a look at this and provide the details needed to make this change before the cluster is created.

Reply:
@dkoshkin - yeah - could you add that? The reason I didn't add this is because we don't have any instrux for `--dry-run` - we could always iterate in a new pr.